inventory/host_vars/_example.com/secrets.example.yml

```diff
@@ -17,40 +13,45 @@ secret_token: 'xxxxxx'
 # Create a Google Maps API key and enable Javascript and Places libraries
 google_maps_api_key:
 
-# Set these if using Amazon S3
+# Mail settings
+mail_host:
+mail_port:
+smtp_username:
+smtp_password:
+
+# Optional mail settings
+## Connection type 'TLS' means STARTTLS usually on port 587
+## Connection type 'SSL' means SSL/TLS usually on port 465
+# mail_secure_connection: 'TLS'
+# mails_from: from_address@example.com
+# mail_bcc: bcc_me@example.com
+
+# Set these if using Amazon S3 for images or backups
 #s3_access_key:
 #s3_secret:
+# Set this if using Amazon S3 for images:
 #s3_images_bucket:
-#s3_region:
+# Set this if using Amazon S3 for backups:
 #s3_backups_bucket:
+# Optional settings for Amazon S3 for images:
+#s3_headers:
+#s3_protocol:
+# Optional settings for Amazon S3 for backups:
 #s3_backups_region:
 
-# Stripe Connect API keys
+# Stripe Connect API keys, set these if you are using Stripe
 # Find these under 'API keys' and 'Connect' in your Stripe account dashboard -> Account Settings
 # Under 'Connect', the Redirect URI should be set to https://YOUR_SERVER_URL/stripe/callbacks (e.g. https://openfoodnetwork.org.uk/stripe/callbacks)
 # Under 'Webhooks', you should set up a Connect endpoint pointing to https://YOUR_SERVER_URL/stripe/webhooks e.g. (https://openfoodnetwork.org.uk/stripe/webhooks)
 # Different keys can be made for testing or live environments
-stripe_client_id: # ca_xxxx
-stripe_instance_secret_key: # sk_xxxx
-stripe_instance_publishable_key: # pk_xxxx
-stripe_endpoint_secret: # whsec_xxxx
+#stripe_client_id: # ca_xxxx
```

Comment:
I think that this will fail because `application.yml.j2` doesn't have `if ... defined` guard clauses for these.

Reply:
I think this is ok because they are defined as empty in roles/app/defaults/main.yml 
I just tested it in staging katuma by removing all stripe keys from the secrets file, this is how it ends up in the server:
![image](https://user-images.githubusercontent.com/1640378/91728989-c96c5600-eb9b-11ea-87f7-f47021c70422.png)
:+1: